libs/cart/state/src/facades/cart/cart-facade.interface.ts

```diff
@@ -232,7 +232,31 @@ export interface DaffCartFacadeInterface<
 	getCompositeCartItemOptions(itemId: U['item_id']): Observable<DaffCompositeCartItemOption[]>;
 	isCartItemOutOfStock(itemId: U['item_id']): Observable<boolean>;
 	/**
-	 * The state of a cart item.
+   * The state of a cart item.
 	 */
   getCartItemState(itemId: U['item_id']): Observable<DaffCartItemStateEnum>;
+  /**
+   * Selects the specified item's price.
+   * Zero by default.
+   */
+	getCartItemPrice(itemId: U['item_id']): Observable<number>;
+  /**
+   * Selects the specified item's quantity.
+   * Zero by default.
+   */
+	getCartItemQuantity(itemId: U['item_id']): Observable<number>;
+  /**
+   * Selects the specified item's row total.
+   * Zero by default.
+   */
+	getCartItemRowTotal(itemId: U['item_id']): Observable<number>;
+  /**
+   * Selects the specified item's array of discounts.
```

Comment:
I think this should specify that these are cart discounts

Reply:
changed